### PR TITLE
Tweak search result list 

### LINKED ISSUE #26: Design search result list
This probably has #24 (plus the addition of other labels) as prerequisite.

Currently, the search result list shows name, some variant names and the GND ID with a link to the entry at the DNB. With this ticket we should discuss and implement a more useful result list. 

Here are some ideas:

- Neither show the GND ID or a link to the DNB service. It suffices to have this in the detail view 
- For `Person`, show occupation (`professionOrOccupation`) and birth/death year.
- For `ConferenceOrEvents`, show place (`placeOfConferenceOrEvent`) and date (`dateOfConferenceOrEvent`. (May not be necessary as this is also appended in parentheses to the preferred name.) 
- for `CorporateBody` show `placeOfBusiness` (more than half of the 1.4 Million corporate bodies have that info).
- For `Work`we definitely need `firstAuthor`/`firstComposer`,
- For `PlaceOrGeographicName` show `geographicAreaCode`
- For `SubjectHeading` show `gndSubjectCategory`

### PR DESCRIPTION
**Pull request by fsteeg**
Will resolve #26

### CODE REVIEW

File: test/ttl/103059083.ttl [hunk 1]
```diff
@@ -0,0 +1,56 @@
+@prefix schema: <http://schema.org/> .
```
Comment (dr0i) [inline test/ttl/103059083.ttl]: Is this file used anywhere? I couldn't find it.

Comment (dr0i) [reply test/ttl/103059083.ttl]: Discussed offline - this file is used to make manually tests.

**Comment by dr0i** [comment]
+1

**Comment by fsteeg** [comment]
Merged with https://github.com/hbz/lobid-gnd/commit/641524bee2cadebaa6ffc3209ee4097367cb6dc0, closing.

**Pull request closed without merging**